db/db_basic_test.cc

```diff
@@ -2101,6 +2095,52 @@ class DBBasicTestWithTimestampWithParam
   }
 };
 
+class DBBasicTestWithTimestamp : public DBBasicTestWithTimestampBase {
```

Comment:
Maybe it's better to let `DBBasicTestWithTimestamp`, `DBBasicTestWithTimestampWithParam` have different db directory. To achieve this, the constructor of `DBBasicTestWithTimestampBase ` can take directory path as argument.
```
class DBBasicTestWithTimestampBase ... {
  DBBasicTestWithTimestampBase(const std::string& dbname);
};
..
class DBBasicTestWithTimestamp : public DBBasicTestWithTimestampBase {
  DBBasicTestWithTimestamp(): DBBasicTestWithTimestampBase("/db_basic_test_with_timestamp") {}
};
...
class DBBasicTestWithTimestampWithParam : 
{
  DBBasicTestWithTimestampWithParam(): DBBasicTestWithTimestampBase("/db_basic_test_with_timestamp") {}
};
```

Reply:
Pushed a new change to use different db directories.